packages/compass-components/src/hooks/use-hotkeys.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useHotkeys as useGlobalHotkeys } from 'react-hotkeys-hook';
+import { InlineKeyCode, css, spacing } from '../';
+
+const shortcutContainerStyles = css({
+  display: 'flex',
+  gap: spacing[1],
+});
+
+const isMac = () => navigator.userAgent.indexOf('Mac') !== -1;
+
+type GlobalHotkeysArgs = Parameters<typeof useGlobalHotkeys>;
+
+/**
+ *
+ * The modifier keys are interpreted differently on macOS and Windows/Linux.
+ * useGlobalHotkeys hook normalizes the modifier keys so that they are interpreted on both platforms as follows:
+ *
+ * Modifier | macOS       | Windows/Linux
+ * -------- | ----------- | -------------
+ * ctrl     | control     | ctrl
+ * shift    | shift       | shift
+ * alt      | option      | alt
+ * meta     | command     | ctrl
+ *
+ */
+export const useHotkeys = (
+  key: string,
+  callback: GlobalHotkeysArgs[1],
+  options?: GlobalHotkeysArgs[2],
+  deps?: GlobalHotkeysArgs[3]
+) => {
+  return useGlobalHotkeys(key, callback, options, deps);
+};
+
+export const mapHotkeyToShortcut = (key: string) => {
+  // map the modifier keys to the platform specific keys
+  const shortcut = isMac()
+    ? key.replace(/\bmeta\b/, '⌘').replace(/\balt\b/, 'option')
```

Comment:
Something makes me think that `useGlobalHotkeys` is accepting different "cases", like probably `Meta` would also work, given `ArrowUp` does, should we cover that too in this regex?